src/index.html

```diff
@@ -4,9 +4,301 @@
+        <article class="catalog__card card" >
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27inch Retina 5K Monoblock (MNED2UA/A)"
+          >
+          <h2 class="card__heading">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__cod">
+            Product code: 195434
+          </p>
+          <section class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">Reviews: 5</p>
+          </section>
+          <section class="card__price">
+            <p class="card__price-label">Price:</p>
+            <p class="card__price-value">$2,199</p>
+          </section>
+          <a href="#buy" class="card__buy-button"> Buy</a>
+        </article>
+
+        <article class="catalog__card card" >
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27inch Retina 5K Monoblock (MNED2UA/A)"
+          >
+          <h2 class="card__heading">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__cod">
+            Product code: 195434
+          </p>
+          <section class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">Reviews: 5</p>
+          </section>
+          <section class="card__price">
+            <p class="card__price-label">Price:</p>
+            <p class="card__price-value">$2,199</p>
+          </section>
+          <a href="#buy" class="card__buy-button"> Buy</a>
+        </article>
+
+        <article class="catalog__card card" >
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27inch Retina 5K Monoblock (MNED2UA/A)"
+          >
+          <h2 class="card__heading">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__cod">
+            Product code: 195434
+          </p>
+          <section class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">Reviews: 5</p>
```

Comment:
checklist p2. go through the file and add some blank lines between blocks

Reply:
fixed